addons/l10n_cl/views/report_invoice.xml

```diff
@@ -216,6 +216,22 @@
             <span t-out="line_amounts['line_description']" t-options="{'widget': 'text'}"/>
         </xpath>
 
+        <xpath expr="//div[contains(@class, 'invoice_main')]" position="inside">
+            <t t-if="show_details_box">
+                <div style="position: fixed; width: 45%; right: 3%;">
+                    <div style="border: 3px solid black; padding: 1rem;" >
+                        <div style="font-size: 1.5rem"> Nombre : ___________________________________ </div>
```

Comment:
no need for the _____ normally the space should be given by the css not the text :smile: